app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -1841,11 +1841,11 @@ BrowserWorld::TickImmersive() {
             DrawImmersive(aEye);
         };
       }
+      m.frameEndHandler = [=]() {
+        m.device->EndFrame(aDiscardFrame ? DeviceDelegate::FrameEndMode::DISCARD : DeviceDelegate::FrameEndMode::APPLY);
+        m.blitter->EndFrame();
+      };
```

Comment:
Hmm this is not correct, we should set the handler even when frameahead is not supported

Reply:
Thanks for the comment, I will take a look more.

Reply:
Rollback the change in this file. Thanks!